docs/concepts/vault-ops-request/overview.md

```diff
@@ -10,4 +10,195 @@ menu_name: docs_{{ .version }}
 section_menu_id: concepts
 ---
 
-# Vault Ops Request Overview concepts
\ No newline at end of file
+> New to KubeVault? Please start [here](/docs/README.md).
+
+# VaultOpsRequest
+
+## What is VaultOpsRequest
+
+`VaultOpsRequest` is a Kubernetes `Custom Resource Definitions` (CRD). It provides a declarative configuration for `Vault` administrative operations like restart, reconfigure TLS etc. in a Kubernetes native way.
+
+## VaultOpsRequest CRD Specifications
+
+Like any official Kubernetes resource, a `VaultOpsRequest` has `TypeMeta`, `ObjectMeta`, `Spec` and Status sections.
+
+Here, some sample `VaultOpsRequest` CRs for different administrative operations is given below:
+
+### Sample `VaultOpsRequest` for restarting `VaultServer`:
+
+```yaml
+apiVersion: ops.kubevault.com/v1alpha1
+kind: VaultOpsRequest
+metadata:
+  name: restart-vault-request
+  namespace: demo
+spec:
+  restart: {}
+  type: Restart
+  vaultRef:
+    name: vault
+status:
```

Comment:
Should we provide the status section here? Or just the `yaml` the user should apply to do restart ops request.